mypy/test/testpep561.py

```diff
@@ -29,6 +29,19 @@ def check_mypy_run(cmd_line: List[str],
     assert returncode == expected_returncode, returncode
 
 
+def is_in_venv() -> bool:
+    """Returns whether we are running inside a venv.
+
+    Based on https://stackoverflow.com/a/42580137.
+
+    """
+    if hasattr(sys, 'real_prefix'):
+        return True
+    else:
+        # https://github.com/python/typeshed/pull/2047
```

Comment:
I merged that typeshed PR, but yet another typeshed sync is also needed.

Reply:
I was expecting that we'd just merge this to get mypy master back in a good state. I can send another small PR to remove the type ignore later.

Reply:
(Actually I propose not to cp that onto the release branch, since the `# type: ignore` below silences the error. We can fix things on the master branch.)